types/repo/AnonymousMessageRepo/MessageAnonymouslyData.ts

```diff
@@ -4,6 +4,10 @@ import {Member} from './AnonymousChannelsData';
 export interface MessageAnonymouslyData {
   channel: ChannelData;
   members: Member[];
+  anon_user_info_emoji_name: string | null | undefined;
+  anon_user_info_emoji_code: string | null | undefined;
+  anon_user_info_color_name: string | null | undefined;
+  anon_user_info_color_code: string | null | undefined;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new fields `anon_user_info_emoji_name`, `anon_user_info_emoji_code`, `anon_user_info_color_name`, and `anon_user_info_color_code` have been added to the `MessageAnonymouslyData` interface. However, it's not clear what these fields represent or how they are used. It would be beneficial to add comments explaining their purpose and usage. Also, consider using a nested object for these properties if they are related.

```diff
 export interface MessageAnonymouslyData {
   channel: ChannelData;
   members: Member[];
+  anonUserInfo?: {
+    emojiName: string | null | undefined;
+    emojiCode: string | null | undefined;
+    colorName: string | null | undefined;
+    colorCode: string | null | undefined;
+  };
   appAdditionalData?: {
     rawJson: any;
     message: string;
```
This change groups the anonymous user information into a single optional object, making the code more readable and maintainable.


<!-- This is an auto-generated comment by OSS CodeRabbit -->